src/Init/Omega/Int.lean

```diff
@@ -0,0 +1,174 @@
+/-
+Copyright (c) 2023 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+prelude
+import Init.Data.Int.Order
+
+/-!
+# Lemmas about `Nat` and `Int` needed internally by `omega`.
+
+These statements are useful for constructing proof expressions,
+but unlikely to be widely useful, so are inside the `Std.Tactic.Omega` namespace.
+
+If you do find a use for them, please move them into the appropriate file and namespace!
```

Comment:
A lot of these lemmas already exist downstream in mathlib / std, right? Can you add a sentence indicating that for many users, they can find the lemmas there?

Reply:
I don't think they do, actually. I initially wrote these when `omega` lived in Mathlib! (on a branch)